MANIFEST.in

```diff
@@ -5,8 +5,8 @@ include dev-requirements.txt
 include versioneer.py
 include src/alembic.ini
 include src/fidesops/_version.py
-graft src/migrations
-exclude src/migrations/README
-exclude src/migrations/script.py.mako
+graft src/fidesops/migrations
+exclude src/fidesops/migrations/README
+exclude src/fidesops/migrations/script.py.mako
```

Comment:
I don't think these are needed now that the migrations are in the `fidesops` directory.  I tested with and without the lines by running the following and the results looked the same.

```py
python -m setup.py sdist --dry-run | grep migrations
```